E2_ESRGAN/lib/model.py

```diff
@@ -6,7 +6,7 @@ class RRDBNet(tf.keras.Model):
   def __init__(
```

Comment:
Re: Your import style at the top: Try not to use `import *` kind of imports. It pollutes the namespace and can cause surprising behavior.